src/Components/ProdDetail/ProductPictures/ProductPictures.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import './productPictures.scss';
+
+class ProductPictures extends Component {
+  constructor() {
+    super();
+    this.state = {
+      pictureList: [],
+      mainPictureSrc: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/ProdDetail/prodDetailData.json')
+      .then(res => res.json())
+      .then(data => {
+        const pictureArr = data.result.detail_image;
+        this.setState({
+          pictureList: pictureArr,
+          mainPictureSrc: pictureArr[0],
+        });
+      });
+  }
+
+  changePicture = e => {
+    this.setState({
+      mainPictureSrc: e.target.src,
+    });
+  };
+
+  render() {
+    const { mainPictureSrc, pictureList } = this.state;
+    return (
+      <div className="ProductPictures">
+        <div className="mainPic">
+          <img alt="product" src={mainPictureSrc} />
+        </div>
+        <div className="smallPic">
+          {pictureList.map(picture => {
+            return (
+              <img
+                className={mainPictureSrc === picture && 'picturefocusOn'}
+                alt="product"
+                key={picture.index}
+                src={picture}
```

Comment:
map 인자로 받는 콜백 함수의 두번째 인자가 index라서 아래와 같이 할 수 있습니다.
그리고 map의 key로 index를 잘 쓰지 않는데 그 이유에 대해서도 알아보면 좋을 것 같습니다.
```suggestion
          {pictureList.map((picture, index) => {
            return (
              <img
                className={mainPictureSrc === picture && 'picturefocusOn'}
                alt="product"
                key={index}
                src={picture}
```

Reply:
두 번째 인자 index로 넣어 수정했습니다. 
배열의 index를 key로 넣을 경우 배열이 재배열 된다면 컴포넌트가 re-rendering 되고 항목의 순서가 바뀌었을 때 key가 바뀌면서 state 순서가 엉망이 될 수 있어 쓰지 않는 점 확인 했습니다! 감사합니다~